### PR TITLE
Configure long polling fallback

### PR DESCRIPTION
**Pull request by fdore**
display connection type in footer

#82 #83

### CODE REVIEW

File: src/client/src/services/model/connectionType.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default {
```
Comment (KeithWoods) [inline src/client/src/services/model/connectionType.js]: @raybooysen do you have comments on this 'enum' styling as opposed to an object with static props?

Am I right in thinking this encourages the use of magic strings and should be avoided?

File: src/client/src/system/service/autobahnConnectionProxy.js [hunk 2]
```diff
@@ -9,11 +9,23 @@ export default class AutobahnConnectionProxy {
...
+    const useSecure = location.protocol === 'https:';
+    const securePort = 443;
+    const defaultPort = 8080;
     this.connection = new autobahn.Connection({
-      url: url,
       realm: realm,
       use_es6_promises: true,
-      max_retries: -1 // unlimited retries
+      max_retries: -1, // unlimited retries,
+      transports: [
```
Comment (KeithWoods) [inline src/client/src/system/service/autobahnConnectionProxy.js]: Bit surprised to only see 2 options here. Were there more than 2 options in the default list?

**Comment by KeithWoods** [comment]
Nice job. Minor question for ray but looks really good.

File: src/client/src/system/service/autobahnConnectionProxy.js (hunk 2 above)
Comment (fdore) [inline src/client/src/system/service/autobahnConnectionProxy.js]: According to the Autobahn documentation, 

> supported transports are WebSocket and HTTP long-poll

cf. http://autobahn.ws/js/reference.html#connection-options

Comment (KeithWoods) [inline src/client/src/system/service/autobahnConnectionProxy.js]: ok fair enough.

**Pull request merged**